extension/chrome/elements/compose-modules/formatters/signed-msg-mail-formatter.ts

```diff
@@ -34,12 +34,14 @@ export class SignedMsgMailFormatter extends BaseMailFormatter {
       const allContacts = [...newMsg.recipients.to || [], ...newMsg.recipients.cc || [], ...newMsg.recipients.bcc || []];
       ContactStore.update(undefined, allContacts, { last_use: Date.now() }).catch(Catch.reportErr);
       const body = { 'text/plain': signedData };
-      return await SendableMsg.create(this.acctEmail, { ...this.headers(newMsg), body, atts });
+      //return await SendableMsg.create(this.acctEmail, { ...this.headers(newMsg), body, atts });
+      return await SendableMsg.createOpenPGPNoDraft(this.acctEmail, { ...this.headers(newMsg), body, atts });
```

Comment:
This should use `createPgpInline` - it's in-lining the signed text into the body

Reply:
ok